nucypher/characters/chaotic.py

```diff
@@ -377,6 +411,22 @@ def __transfer(self, disbursement: int, recipient_address: str) -> str:
                                             target_address=recipient_address,
                                             sender_address=self.checksum_address)
         txhash = receipt['transactionHash']
+        if self.distribute_ether:
+            ether = self.ETHER_AIRDROP_AMOUNT
+            transaction = {'to': recipient_address,
+                           'from': self.checksum_address,
+                           'value': ether,
+                           'gasPrice': self.blockchain.client.gas_price}
+            ether_txhash = self.blockchain.client.send_transaction(transaction)
+
+            self.log.info(f"Disbursement #{self.__disbursement} OK | NU {txhash.hex()[-6:]} | ETH {ether_txhash.hex()[:6]} "
```

Comment:
@vepkenez out of curiosity - we print the last 6 hex instead of the first 6?

Reply:
@derekpierre I didn't write this originally.  I don't have an answer.  @KPrasch?

Reply:
Heh - that's inconsistent.  Feel free to change it.

Reply:
Note that there are a few other spots in the code.

Reply:
Am I going with the 1st 6 or the last 6?
What is the standard we want?  It's almost a 50/50 split in occurrences....

Reply:
My assumption would be first 6 - easier to identify. What do you think @KPrasch ?